Doc/whatsnew/3.13.rst

```diff
@@ -189,6 +189,14 @@ typing
   check whether a class is a :class:`typing.Protocol`. (Contributed by Jelle Zijlstra in
   :gh:`104873`.)
 
+
+pdb
+---
+
+* Add ability to move between chained exceptions during post mortem debugging in :func:`pm()` using
```

Comment:
There's a warning here about the func:pm reference. I don't think these references work in what's new. Maybe @CAM-Gerlach can advise.

Reply:
You don't need the brackets in the `:func:` markup; they're added by sphinx when the markup is rendered. You also need to tell sphinx the module where the function is from, or sphinx can't add a link to it.

```suggestion
* Add ability to move between chained exceptions during post mortem debugging in :func:`~pdb.pm` using
```

Reply:
Thanks !